README.md

```diff
@@ -119,6 +119,17 @@ mergeStream(project.sources(), project.dependencies())
   .pipe(gulp.dest('build/'));
 ```
 
+You may also provide an options object as and argument to the `bundler()` method to fully customize how bundling works in your build pipeline.  These include all of the `polymer-bundler` constructor arguments, `analyzer`, `excludes`, `inlineCss`, `inlineScripts`, `rewriteUrlsInTemplates`, `sourcemaps`, and `stripComments`, as well as `strategy` and `urlMapper` arguments used when invoking `generateManifest()`.
```

Comment:
I'm very tempted to rewrite bundler to make `urlMapper` and `strategy` part of the constructor anyways, so losing the fidelity here isn't too bad for now.